frontend/src/app/types/app/app-state.ts

```diff
@@ -30,4 +30,5 @@ export interface AppState {
 
   featureFlags: Array<FeatureFlagState>;
   activeTab: string;
+  activeComplaintsViewType: "list" | "map";
```

Comment:
I think this needs to be defined in a redux migration or users won't get it without a hard cache clear due to redux persist.